supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -65,45 +65,12 @@
 
 *See also*: xref:read-v[read (verb)]
 
-[discrete]
-[[real-time-n]]
-==== real time (noun)
-*Description*: As a noun, "real time" is the actual time during which something takes place, for example, "The computer may partly analyze the data in real time (as it comes in) -- R. H. March."
-
-*Use it*: yes
-
-*Incorrect forms*: realtime
-
-*See also*: xref:real-time-adj[real-time]
-
-[discrete]
-[[real-time-adj]]
-==== real-time (adjective)
-*Description*: As an adjective, use the hyphenated form, for example, "XEmacs is a self-documenting, customizable, extensible, real-time display editor."
-
-*Use it*: yes
-
-*Incorrect forms*: realtime
-
-*See also*: xref:real-time-n[real time]
-
-[discrete]
-[[reboot]]
-==== reboot (verb)
-*Description*: "Reboot" means to turn off a computer's operating system and then turn it on.
-
-*Use it*: yes
-
-*Incorrect forms*: re-boot
-
-*See also*:
-
 [discrete]
 [[recommend]]
 ==== recommend (verb)
 *Description*: Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action. This allows Red Hat to be more prescriptive in documentation and prevent any user uncertainty, and is easier for upstream/downstream coordinated efforts.
 
-For example, instead of "Red Hat recommends using X package because", write "Use this package because" or "Use this package when". 
+For example, instead of "Red Hat recommends using X package because", write "Use this package because" or "Use this package when".
```

Comment:
A terminal period typically goes inside the closing quote mark.

Reply:
@leswilliams44 Per my earlier comment, this PR is only removing content and moving it. I didn't want to make any edits at all within this PR. So any other edits you find, can you submit them as a separate PR, like you did with #66?